zokrates_core/src/compile.rs

```diff
@@ -249,6 +249,8 @@ fn check_with_arena<'ast, T: Field, E: Into<imports::Error>>(
     let typed_ast = Checker::check(compiled)
         .map_err(|errors| CompileErrors(errors.into_iter().map(CompileError::from).collect()))?;
 
+    log::trace!("\n{}", typed_ast);
```

Comment:
probably a leftover that should be removed?

Reply:
No, it's useful to have a trace of the AST post semantic checker, before we hit the static analyser.